src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  width: 200px;
+}
+
+.product {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+}
+
+.product__card {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto;
+}
+
+.name {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin: 40px auto 0;
+}
+
+.code {
+  margin: 4px 0 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+}
+
+.group {
+  display: flex;
+  justify-content: space-between;
+}
+
+.group--top {
+  margin-top: 16px;
+}
+
+.group__stars {
+  display: flex;
+}
+
+.group__star {
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.group__reviews {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: right;
+  height: 16px;
+  display: flex;
+  align-items: center;
+}
+
+.group__stars .group__star:nth-child(5) {
+  background-image: url(../images/star.svg);
+}
+
+.prace {
+  margin-top: 24px;
+  height: 18px;
+  justify-content: space-between;
+}
+
+.prace__1 {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-family: Roboto, sans-serif;
```

Comment:
elements can inherit the font family. No need to add this style for every element